[EDU-645] Data Explorer multiple files (#641)
seqeralabs/docs: platform-cloud/docs/data/data-explorer.md

## Upload files to private buckets

Data Explorer supports single or bulk file uploads to your private cloud storage buckets. From the **View cloud bucket** page, select **Upload** and choose either the **Upload files* or **Upload folder** option. You can also drag and drop files and folders directly into Data Explorer. You can upload up to 300 files at a time via the Platform interface. The file size upload limits reflect the size limitations of the relevant cloud storage provider. 

Currently, these limits are:

- [AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html)
  - Single `PUT` upload: 5 GB
  - Multi-part upload: 5 TB
  
- [Azure](https://learn.microsoft.com/en-us/rest/api/storageservices/put-blob?tabs=microsoft-entra-id#remarks)
  - Single `PUT` upload: 5 GB
  - Multi-part upload: 4.77 TB
  
- [GCP](https://cloud.google.com/storage/quotas#objects):
  - Single `PUT` upload: 5 TB
  - Multi-part upload: 5 TB

> To cancel an upload, select **X** in the upload window. Any files not uploaded will display as **Failed**. Files that were successfully uploaded will not be removed. 

Reviewer: Sounds good.

## Upload files to private buckets

> Data Explorer supports file uploads to your private cloud storage buckets. From the **View cloud bucket** page, select **Upload file** to select the files you wish to upload. You can upload up to 300 files at a time via the Platform interface. 

Reviewer: ```suggestion
Data Explorer supports single or bulk file uploads to your private cloud storage buckets. From the **View cloud bucket** page, select **Upload** and choose either the **Upload files* or **Upload folder** option. You can also drag and drop files and folders directly into Data Explorer to upload directly. You can upload up to 300 files at a time via the Platform interface. File size upload limits reflect the size limitations of the relevant cloud storage provider. Currently, these limits are:

- [AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html)
  - Single PUT upload: 5GB
  - Multi-part upload: 5TB
- [Azure](https://learn.microsoft.com/en-us/rest/api/storageservices/put-blob?tabs=microsoft-entra-id#remarks)
  - Single PUT upload: 5GB
  - Multi-part upload: 4.77TB
- [GCP](https://cloud.google.com/storage/quotas#objects):
  - Single PUT upload: 5TB
  - Multi-part upload: 5TB
```

> To cancel the upload, select **X** in the upload window. Any files that were not uploaded will display as **Failed**. Files that were successfully uploaded will not be removed. 

Reviewer: ```suggestion
To cancel an upload, select **X** in the upload modal window. Any files not uploaded will display as **Failed**. Files that were successfully uploaded will not be removed. 
```

:::note

> You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.

Reviewer: ```suggestion
You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Platform. CORS configuration differs for each provider.
```
